.github/.gitignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
Is this to exclude rendered output of Rmds?

Reply:
Yeah, if you have any `.md` files in that directory (it's automatically added by some helper in usethis)